build.gradle

```diff
@@ -82,15 +82,21 @@ tasks.register('acceptance-tests', Test) {
 }
 
 dependencies {
+    // transitive dependencies to solve the vulnerabilities which are part of our dependencies
+    implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.10.0'
```

Comment:
[As suggested by retrofit maintainer](https://github.com/square/retrofit/discussions/3753#discussioncomment-3765933)

Reply:
So this should be sufficient to have the newer version picked up instead of the one provided by retrofit?

Reply:
Correct, we can verify by: 
```
./gradlew dependencies | grep okhttp:3.14.9 -B 1

+--- com.squareup.retrofit2:retrofit:2.9.0
|    \--- com.squareup.okhttp3:okhttp:3.14.9 -> 4.10.0 (*)
```